xbmc/addons/AddonManager.cpp

```diff
@@ -440,6 +465,11 @@ bool CAddonMgr::GetAddonUpdateCandidates(VECADDONS& updates) const
   return updates.empty();
 }
 
+const std::vector<std::string>& CAddonMgr::GetTrustedAddonRepos() const
```

Comment:
is this used? Why does it need to be public?

Reply:
actually not used, but might be needed at a later point. fine if it should be removed for now.

Reply:
Yes, please remove. Never add code until it is required. As you dig deeper into the codebase you will comes across lots of misleading functions that don't make sense cause they were added for a future feature that never got implemented.

Reply:
I'd say to keep it simple and remove anything that is not used now. If it is needed later, it can always be added later :)

Reply:
on the way to the trash bin 😉